news-explorer-frontend/src/components/SearchForm/SearchForm.jsx

```diff
@@ -0,0 +1,44 @@
+import React, {useState} from 'react';
+import './SearchForm.css';
+import {data} from '../../utils/data.js';
+
+function SearchForm({onSearchSubmit, setKeyword,  }) {
+ const [searchInput, setSearchInput] = useState("");
+
+  function handleSearchSubmit(e) {
+    e.preventDefault(e);
+    setKeyword(searchInput);
+  }
+
+  function handleChange(e) {
+    setSearchInput(e.target.value);
+  }
+
+  return(
+    <section className="search-form__content">
+    <div className="search-form__content-top">
+      <h1 className="search-form__content-title">What's going on in the world?</h1>
+      <p className="search-form__content-subtitle">Find the latest news on any topic and save them in your personal account.</p>
+      <form className="search-form__content-form"  onSubmit={handleSearchSubmit} >
+        <label className="search-form__content-form-field">
+          <input 
+            className="search-form__content-form-input" 
+            type="text" 
+            placeholder="Enter Search Topic"  
+            onChange={handleChange} // onChange={(e) => searchItems(e.target.value) && handleChange(e)}
+            value={searchInput || ''}
+            id="search-input" name="search-input"
+            required
+          />
+          <button type="submit" className="search-form__content-form_submit"  onClick={onSearchSubmit} >
```

Comment:
`onClick={onSearchSubmit}` 


please, delete it from the submit button


loading effect should be inside `handleSearchSubmit`

you shouldn't add `clicks` on submit buttons because they are created specially for launching `submit` events

Reply:
cleared the extra eventListener